algosdk/transaction.py

```diff
@@ -2507,7 +2507,9 @@ class LogicSig:
     """
     Represents a logic signature
 
-    NOTE: This type is deprecated. Use LogicSigAccount instead.
+    NOTE: LogicSig is still safe for escrow and some other use cases.
+    But it is not fully compatible with delegated logicsig use cases, e.g., rekeying.
+    Under that circumstance, it is recommended to use LogicSigAccount in that case.
```

Comment:
@ahangsu Optional - I don't feel strongly, but I think the docstring should read more objectively.   

I realize we're describing functionality where we're a bit unfamiliar with the details.  If you don't feel comfortable with the _suggestion_, then feel welcomed to keep as is.

```suggestion
    NOTE: LogicSig is safe for non-rekeying usage.  If using rekeyed accounts, use LogicSigAccount instead.
```